johnsonandjohnson/Bodiless-JS
# docs(vital-demo-next, bodiless-webpack): Fix vital-demo-next docs

## Changes

### Missing Docs in Vital Demo Next Site

Add packages to the `vital-demo-next` site's `package.json` file, filling in the gaps in the documentation.

See below for breakdown of added packages and related documentation sections/pages:

#### Vital Design System

- `@bodiless/vital-examples`
  - [Vital Design System > Curriculum](http://localhost:3000/#/VitalDesignSystem/Curriculum/)

#### Components

- `@bodiless/bv`
  - [Components > Bv](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/bv)
- `@bodiless/card`
  - [Components > Card](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Card)
- `@bodiless/carousel`
  - [Components > Carousel](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Carousel)
- `@bodiless/curator`
  - [Components > Social Wall](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SocialWall)
- `@bodiless/ga4`
  - [Components > Analytics](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Analytics/)
- `@bodiless/navigation`
  - [Components > Navigation](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Navigation/)
- `@bodiless/oidc`
  - [Components > OIDC](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/OIDC)
- `@bodiless/organisms`
  - [Components > Embed](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Embed)
  - [Components > Filter By Group](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/FilterByGroup)
  - [Components > Single Accordion](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SingleAccordion)
  - [Components > Social Share](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/SocialShare)
- `@bodiless/richtext`
  - [Components > Editors](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Editors/)
- `@bodiless/search`
  - [Components > Search](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Search/)
- `@bodiless/table`
  - [Components > Table](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/Table)
- `@bodiless/youtube`
  - [Components > You Tube](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/YouTube)

#### Tools

- `@bodiless/accessibility`
  - [Tools > Accessibility](https://johnsonandjohnson.github.io/Bodiless-JS/#/Tools/Accessibility/)

#### Development

- `@bodiless/psh`
  - [Development > Web Platforms > Platform.sh](https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/WebPlatforms/Platform.sh)
- `@bodiless/gatsby-theme-bodiless`
  - [Development > Gatsby Theme](https://johnsonandjohnson.github.io/Bodiless-JS/#/Development/GatsbyTheme)

### Remove Redundant Bodiless CLI Page

- Update `@bodiless/webpack` README file to no longer be an outdated clone of the "Bodiless CLI" page.
- Remove `@bodiless/webpack` README from being added to Docsify.
  - Remove Docsify-related files.
  - Remove `@bodiless/cli` from dependencies, as only used for Docsify.

----

## Test Instructions

01. Run the docs for the `vital-demo-next` site:
    `cd sites/vital-demo-next && npm run docs`
02. Go to <http://localhost:3000/#/>.
03. Verify that documentation matches "Live" documentation: <https://johnsonandjohnson.github.io/Bodiless-JS/#/>.
